src/docs/Components/Hero/previews/box.preview.scss

```diff
@@ -0,0 +1,27 @@
+@import 'settings-tools/_all-settings';
```

Comment:
Since this is not going to be committed now, can you please remove this file from the commit?